### PR TITLE
Add AzureEnvironment to ILabAccount

### PR DESCRIPTION
**Pull request by rpdome**
Related PR: https://github.com/AzureAD/ad-accounts-for-android/pull/2553

### CODE REVIEW

File: LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/LabConstants.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ static final class FederationProvider {
         public static final String CIAM = "ciam";
     }
 
-    static final class AzureEnvironment {
+    public static final class AzureEnvironment {
```
Comment (shahzaibj) [inline LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/LabConstants.java:97]: Was this change actually needed?

**Review by shahzaibj — APPROVED** [review]

File: LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/LabConstants.java (hunk 1 above)
Comment (rpdome) [reply LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/LabConstants.java:97]: Yes. In the other PR, I used the AzurePPE constant to check (against AzureEnvironment of the account) if we should be making a drs discovery request to prod or ppe.

**Review by p3dr0rv — APPROVED** [review]
